src/redux/books/books.js

```diff
@@ -1,32 +1,128 @@
+/* eslint-disable camelcase */
+import axios from 'axios';
+
 // Actions Types
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const LOAD_BOOKS_IN_PROGRESS = 'LOAD_BOOKS_IN_PROGRESS';
+const LOAD_BOOKS_SUCCESS = 'LOAD_BOOKS_SUCCESS';
+const LOAD_BOOKS_FAILURE = 'LOAD_BOOKS_FAILURE';
 
-// Reducer
+// Reducers
```

Comment:
Please remove this comment and any other comments in the codebase. Comments make our codebase look ugly.

Reply:
Comments are very useful to illustrate the job of each code block. 

> These are called Clarification comments
> Clarification comments are intended for anyone (including your future self) who may need to maintain, refactor, or extend your code.

 I suggest having a look here https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/